frontend/portals/ItemListAfter/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
I'd suggest rename this file to CartItemListAfter so it's obvious that it's only rendered in the cart page

Reply:
Renamed